requirements.txt

```diff
@@ -2,6 +2,13 @@ python>=3.11.0, <4.0.0
 requests>=2.31.0, <3.0.0
 caf.toolkit>=0.2.1, <0.3.0
 sqlalchemy>=2.0.0, <3.0.0
+psycopg2
 tqdm>=4.66.0, <5.0.0
-gtfs-realtime-bindings>=1.0.0, <2.0.0
 pydantic>=1.10.0, <2.0.0
+# Required for converting AVL from lat / lon to easting / northing
+pyproj>=3.6.0, <4.0.0
+
+# Packages not available on conda-forge, only available on PyPI
```

Comment:
'requirements.txt' files don't give any information around what package manager to use (conda-forge vs pip), there are other file formats which do. The 'requirements.txt' file just provides package names and versions (and in some cases a URL instead of version).

In this case if you attempt to install all packages with conda it will fail due to not finding the gtfs package, I have been commenting out that line, installing using conda then installing the final package using pip. I think it might be clearer if we split the requirements file into 2:
- 'requirements.txt': containing all packages available from pip or conda
- 'requirements-pip.txt': containing the packages which are only available from pip

Then creating an environment with conda could be done with the following 2 commands:
- `conda create -n env_name --file requirements.txt` to create the env and install the packages available on conda
- `pip install -r "requirements-pip.txt"` to install the remaining pip only packages

This could be done with pip (or pipenv) in a single command: `pip install -r requirements.txt -r requirement-pip.txt`

Reply:
Split into 2 requirement files in commit f39e0bb6da74409e35fb6e89593941fb07d76fd0